### PR TITLE
grafana-cli: add page

### PR DESCRIPTION
**Pull request by hellojukay**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/grafana-cli.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# grafana-cli
+
+> A small executable that is bundled with Grafana serve.
+> More information: <https://grafana.com>.
+
+- Install a grafana plugin:
```
Comment (EmilySeville7cfg) [inline pages/linux/grafana-cli.md:6]:
```suggestion
- Install/update/remove specific plugins:
```

This page already describes `grafana-cli`, it's not needed to mention it explicitly.

File: pages/linux/grafana-cli.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`grafana-cli plugins install "{{plugin-id}}"`
```
Comment (EmilySeville7cfg) [inline pages/linux/grafana-cli.md:8]:
```suggestion
`grafana-cli plugins {{install|update|remove}} {{plugin_id1 plugin_id2 ...}}`
```

If you know that plugin id doesn't contain spaces in it or documentation states it explicitly please remove quotes.

File: pages/linux/grafana-cli.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- List installed plugins:
+
+`grafana-cli plugins ls`
+
+- Remove one plugin:
+
+`grafana-cli plugins remove "{{plugin-id}}"`
```
Comment (EmilySeville7cfg) [inline pages/linux/grafana-cli.md:16]: This example can be joined with the first one.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/grafana-cli.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# grafana-cli
+
+> A small executable that is bundled with Grafana serve.
+> More information: <https://grafana.com>.
```
Comment (marchersimon) [inline pages/linux/grafana-cli.md:4]:
```suggestion
> More information: <https://grafana.com/docs/grafana/latest/cli/>.
```

File: pages/linux/grafana-cli.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+- Install/update/remove specific plugins:
```
Comment (marchersimon) [inline pages/linux/grafana-cli.md:6]:
```suggestion
- Install, update, or remove specific plugins:
```

File: pages/linux/grafana-cli.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+`grafana-cli plugins {{install|update|remove}} {{plugin_id1 plugin_id2 ...}}`
+
+- List installed plugins:
```
Comment (marchersimon) [inline pages/linux/grafana-cli.md:10]:
```suggestion
- List all installed plugins:
```

**Review by marchersimon — APPROVED** [review]
Thanks @hellojukay

**Review by marchersimon — CHANGES_REQUESTED** [review]
I just noticed that this page should be moved to the `common` directory, as it's also available on OSX and Windows

**Review by marchersimon — APPROVED** [review]
👍🏽

**Pull request merged**